docs/constants/navigation-deprecated.js

```diff
@@ -220,8 +220,8 @@ const general = [
       makePage('ui-programming/react-native-styling-buttons.md'),
     ]),
   ]),
-  makeSection('Expo Module API (Alpha)', [
-    makeGroup('Expo Module API (Alpha)', [
+  makeSection('Expo Modules', [
```

Comment:
maybe we should call this section "Creating Native Modules"? if it's called "Expo Modules" it is too vague i believe

Reply:
I think it has some pros and cons. It might be more clear for newcomers, but on the other side I think we want to populate "Expo Modules" name so people can distinguish it from "React Native modules". Taking that into account, I would leave it as is.

Reply:
that's a good point but i think that's another argument for why we might want to call this "Creating Native Modules" - we should have a page in this section that explains why this exists and how it's different from React Native modules (basically we can pull some stuff out of your app.js conf talk for that)

--

Comment posted using [Unblocked](https://getunblocked.com/dashboard/team/5bf29d2c-23ae-4af8-8ccc-fe767cd2df79/thread/3c9f9bfe-cebd-4a6e-b419-490830438df0?message=07029557-8639-442d-ab84-03679ed1d749).